test/jasmine/tests/image_test.js

```diff
@@ -100,6 +101,82 @@ describe('image supplyDefaults', function() {
     });
 });
 
+describe('image smart layout defaults', function() {
+    var gd;
+    beforeEach(function() {
+        gd = createGraphDiv();
+    });
+
+    afterEach(destroyGraphDiv);
+
+    it('should reverse yaxis if only images are present', function() {
```

Comment:
should say `should reverse yaxis if images are present` i.e. no `only`

Reply:
I'd just :hocho: the `only`. There's a nice progression in complexity in this `declare` block

Reply:
Sweet! Done in https://github.com/plotly/plotly.js/pull/4307/commits/408b46ba5d1bddcd496b9397f1864b0749ecec89